packages/foreman/tfm/tfm.spec

```diff
@@ -309,6 +310,9 @@ selinuxenabled && load_policy || :
 %{_root_sysconfdir}/rpm/macros.%{scl_name}-scldevel
 
 %changelog
+* Tue Apr 28 2020 Tomer Brisker <tbrisker@gmail.com>
```

Comment:
```suggestion
* Tue Apr 28 2020 Tomer Brisker <tbrisker@gmail.com> - 6.0-5
```

Reply:
hmm, odd, i used obal to do the rev bump, strange it didn't add the version here.